gradle/libs.versions.toml

```diff
@@ -6,7 +6,7 @@ composeCompiler = "1.4.7"
 javaTarget = "11"
 jcodec = "0.2.5"
 kotlin = "1.8.21"
-ktlint = "0.46.1"
+ktlint = "0.49.1"
```

Comment:
[Ktlint 0.49.0](https://github.com/pinterest/ktlint/releases/tag/0.49.0) contains some breaking changes to the API, and Paparazzi is behind on 0.46.1 which is incompatible with latest Spotless.